### PR TITLE
Fixed tooltip sizes and positioning for responsive design

### PR DESCRIPTION
**Pull request by AndriiBlagodyr**

### CODE REVIEW

**Comment by AndriiBlagodyr** [comment]
Desktop: 
desktop
Ipad:
laptopLeft
laptopRight
Mobile:
mobile

File: src/components/Sidebar/SidebarBalances.jsx [hunk 1]
```diff
@@ -203,6 +199,34 @@ const TooltipWrapper = styled.div`
 
     border-left: 30px solid ${props => props.theme.colors.bgHighlightBorder};
   }
+
+  @media screen and (max-width: ${responsive.tablet}) {
+    width: 240px;
+    padding: 16px 40px;
+
+    right: ${props => (props.left ? '70px' : '-275px')};
```
Comment (flppv) [inline src/components/Sidebar/SidebarBalances.jsx:207]: Let's not create props with CSS rules-like names.

File: src/components/Sidebar/SidebarBalances.jsx [hunk 2]
```diff
@@ -222,12 +246,14 @@ export const TrimmedApproved = withTrimmedWrapper(({ value }) => (
...
 ));
 
-function Hint({ children }) {
+function Hint({ children, left }) {
   const [active, setActive] = useState(false);
 
   return (
     <BalanceHint onPointerOver={() => setActive(true)} onPointerLeave={() => setActive(false)}>
-      <TooltipWrapper active={active}>{children}</TooltipWrapper>
+      <TooltipWrapper active={active} left={left}>
```
Comment (flppv) [inline src/components/Sidebar/SidebarBalances.jsx:254]: Instead of creating manually extra prop, show this tooltip at the bottom of the icon on tablets

**Review by flppv — COMMENTED** [review]
- Please move tooltips to the bottom on tablets layout (when balances are ontop) 
- Remove extra props
- Make triangles 75% smaller on tablets and 50% smaller on smartphones

**Comment by AndriiBlagodyr** [comment]
Updated Screeshots:
UpdDesktop
UpdLaptop
UpdMobile

**Review by flppv — APPROVED** [review]

**Pull request merged**